packages/server-wallet/package.json

```diff
@@ -84,7 +84,7 @@
     "lint:check": "eslint \"*/**/*.{js,ts}\" --cache",
     "lint:write": "eslint \"*/**/*.{js,ts}\" --fix",
     "precommit": "lint-staged --quiet",
-    "prepare": "rm -rf lib; tsc -b .",
+    "prepare": "rm -rf lib; tsc -b . && node ./scripts/inject-wallet-version.js",
```

Comment:
Here's where we inject the wallet version. It'll only get into the build js files, so won't appear in our ts-node scripts and tests.